packages/ember-template-compiler/lib/plugins/transform-attrs-into-args.js

```diff
@@ -27,6 +27,20 @@ export default function TransformAttrsToProps() {
   this.syntax = null;
 }
 
+function isAttrs(node) {
+  if (node.parts[0] === 'attrs') {
+    return true;
+  }
+
+  let [_this, attrs] = node.parts;
```

Comment:
I removed this just in case. I think I was _safe_ sense I was just reading the values.